### DISCUSSION TITLE
wallet

### DISCUSSION THREAD

**Discussion by 11eren** [IBAX-io/go-ibax#181] [Q&A] [answered]
hi I had created a wallet but I didn't save pirvate key and I couldn't log in then I created a new wallet whether the points I earned remained in the old wallet orhi I had created a wallet but I didn't save pirvate key and couldn't log in then I created a new wallet whether the points I earned remained in the old wallet or if my points came to my new wallet please help me thank you

**Comment by ibaxio** [comment, marked as answer]
Sorry to hear about that, unfortunately we follow the principles of the blockchain so please be careful about saving your SEED or Private Key.

If you have another Twitter account, you can continue to participate in our campaign with your new account.

Thanks.
